tests/e2e/tests/e2e/OpenshiftConnector.spec.ts

```diff
@@ -59,14 +59,14 @@ suite('Openshift connector user story', async () => {
           'location': 'https://github.com/maxura/nodejs-hello-world.git',
           'type': 'git'
         }
-      }
+      },
```

Comment:
There are still some trailing commas here

Reply:
This commas are expected due to the linter changes[shown below]. 
As we mentioned arrays as Always, "," are included. Its good to have [Refer Article : https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8 ]
```
"trailing-comma": [true, {
  "singleline": "never",
  "multiline": {
    "objects": "ignore", 
    "arrays": "always",
    "functions": "never",
    "typeLiterals": "ignore"
  }
}]
```